src/librustc/diagnostics.rs

```diff
@@ -368,6 +368,73 @@ loop. Without a loop to break out of or continue in, no sensible action can be
 taken.
 "##,
 
+E0282: r##"
+This error indicates that type inference did not result in one unique possible
+type, and extra information is required. In most cases this can be provided
+by adding a type annotation. Sometimes you need to specify a generic type
+parameter manually.
+
+A common example is the `collect` method on `Iterator`. It has a generic type
+parameter with a `FromIterator` bound, which is implemented by `Vec` and
+`VecDeque` among others. Consider the following snippet:
+
+```
+let x = (1_i32 .. 10).collect();
```

Comment:
and if you did that, you could avoid talking about `VecDeque`; you could just say that `Vec<char>` and *`String`* are suitable candidates:

```rust
    const HELLO: &'static str = "hello";
    let s: String = HELLO.chars().skip(1).collect();
    println!("Bert says, {:?}", s);
```